services/ResourceServiceProvider/src/Provider/ResourceServiceProvider.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Islandora\ResourceService\Provider;
+
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+use Silex\ControllerProviderInterface;
+use Islandora\Chullo\FedoraApi;
+use Islandora\Chullo\TriplestoreClient;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\Yaml\Yaml;
+use Islandora\ResourceService\Controller\ResourceController;
+
+class ResourceServiceProvider implements ServiceProviderInterface, ControllerProviderInterface
+{
+  /**
+   * Part of ServiceProviderInterface
+   */
+  function register(Application $app) {
+    //
+    // Define controller services
+    //
+    $app['islandora.resourcecontroller'] = $app->share(function() use ($app) {
+      return new \Islandora\ResourceService\Controller\ResourceController($app);
+    });
+    if (!isset($app['twig'])) {
```

Comment:
@whikloj, do you mean the "use" statements? I was unsure if putting there had any logic, but i thought it could be a good idea for code clarity/new users. I can remove it once all others had a look at the code